### DISCUSSION TITLE
General QoL - Hidden/Replays/Scores/Editor

### DISCUSSION THREAD

**Discussion by BMcGraw1215** [ppy/osu#31318] [Ideas]
1. Keep local temporary replays. The current way of having accessible replays is rough, with how manual it is. There are times when a player repeatedly fails a section and doesn't notice until the nth attempt, and then wants to see and compare what's happening. Unless they manually save every replay, they can't. This would be easier if there were a handful of replays being held on file temporarily.
2. Allow purging local scores/replays up until nth score across a library.
3. Give some lead time when playtesting in the editor. pressing play immediately sets you into the map with very little lead time. 
4. A small tutorial or description panes for the editor. Learning exactly how slider tools and other functions work isn't openly intuitive, and takes a bit of guesswork to figure out.

This upcoming section is more so because I'm a complete HD player. I cannot read NM, and can only play the game in a fun capacity in HD. While I know this is a personal "skill issue", I do know other players that strictly play HD, or that try to learn HD for extended periods, so this is applicable in those cases too.
 
1. Editors should let players test with HD on. (or other mods even)
2. A way to temp disable HD in Replays. ([Already suggested](https://github.com/ppy/osu/issues/8357), just reminder)
3. An options setting for default mods to have on when starting the game. 
  -  Maybe this idea could even be expanded to defaulting them on when joining Freemod lobbies?

(Thanks Pishi for pointing me in this direction. :) )

**Comment by peppy** [comment]
> Learning exactly how slider tools and other functions work isn't openly intuitive, and takes a bit of guesswork to figure out.

There are basic tooltips on the editor controls, including slider controls.

> Editors should let players test with HD on. (or other mods even)

A play test mode is planned, the editor is not designed for this purpose.
